website/fiddle/package-lock.json

```diff
@@ -2432,9 +2432,9 @@
       "dev": true
     },
     "fundamental-styles": {
-      "version": "0.23.0",
-      "resolved": "https://registry.npmjs.org/fundamental-styles/-/fundamental-styles-0.23.0.tgz",
-      "integrity": "sha512-A5VtoDI8GebDziLmP4e1S4RXwlagglCgBUfHBCCe7OPzbIWH1s4rQ1mO6rp+Gd8aj0uvIe1mkHi+pNKEdqSg2Q=="
+      "version": "0.18.0",
```

Comment:
this seems to be an old remaining change as well. 
Run npm install in this folder and commit the changes to fix this.